base/standard/src/main/java/org/openscience/cdk/tools/FormatStringBuilder.java

```diff
@@ -60,7 +60,7 @@
  * @cdk.githash
  * @cdk.license BSD
  */
-public class FormatStringBuffer {
+public class FormatStringBuilder {
```

Comment:
I would not recommend changing the name of this public class. That would be an API change. That a `FormatStringBuffer` internally uses a `StringBuilder` is not something to worry about.

Reply:
I'm easy either way, this shouldn't really be part of the public API anyways as it's a helper class for some writers. But... there may be some numpty who has used it in a critical piece of code so yeah I wouldn't rename and leave it public. If anything I would rename it and move to org.openscicence.cdk.io package as then I think it can be non-public?

Reply:
> If anything I would rename it and move to org.openscicence.cdk.io package as then I think it can be non-public?

I like this idea the most.

Two questions:
- which **module** should `FormatStringBuilder` be assigned to? `standard` (as of now) or `io`?
- Should I leave the original / unmodified `FormatStringBuffer` in place and mark it with `@deprecated`. We could remove in 3.0. This way the semver contract would be honored.

Reply:
Yeah I would put in io, OSGi/JigSaw don't like have packages split across multiple modules but I don't mind as it keeps things cleaner IMO.